src/button/types.ts

```diff
@@ -2,13 +2,13 @@ import React from "react";
 import { ComponentLoadingSpinnerProps } from "src/shared/component-loading-spinner/component-loading-spinner";
 
 type StyleType = "default" | "secondary" | "light" | "link";
-
 export interface ButtonProps
     extends React.ButtonHTMLAttributes<HTMLButtonElement> {
     /** The style type of the button. Values: "default" | "secondary" | "light" | "link" */
     styleType?: StyleType | undefined;
     /** Indicates if a loading spinner is to be displayed */
     loading?: boolean | undefined;
+    danger?: boolean | undefined;
```

Comment:
Could you add a jsdoc comment using the same description as the one in `props-table`. This is to allow the description to appear when one hovers the prop in code.

Reply:
applied